examples/spring-boot-with-svcat-on-openshift-example/src/main/java/io/dekorate/example/sbonopenshift/Main.java

```diff
@@ -15,15 +15,14 @@
  */
 package io.dekorate.example.sbonopenshift;
 
-import io.dekorate.kubernetes.annotation.Env;
 import io.dekorate.openshift.annotation.OpenshiftApplication;
 import io.dekorate.servicecatalog.annotation.ServiceCatalog;
 import io.dekorate.servicecatalog.annotation.ServiceCatalogInstance;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 
-@OpenshiftApplication(buildEnvVars = @Env(name="foo", value = "bar"))
+@OpenshiftApplication
```

Comment:
Does this example still make sense without the build env vars?

Reply:
Same as above